src/elements/toniq-dropdown/toniq-dropdown.element.ts

```diff
@@ -0,0 +1,161 @@
+import {assign, css, defineElementEvent, html} from 'element-vir';
+import {ChevronDown24Icon} from '../../icons';
+import {interactionDuration, noUserSelect, toniqFontStyles} from '../../styles';
+import {applyBackgroundAndForeground, toniqColors} from '../../styles/colors';
+import {removeNativeButtonStyles} from '../../styles/native-styles';
+import {defineToniqElement} from '../define-toniq-element';
+import {ToniqIcon} from '../toniq-icon/toniq-icon.element';
+
+declare global {
+    interface Window {
+        dropdownListenerAdded: boolean;
+    }
+}
+window.dropdownListenerAdded = window.dropdownListenerAdded || false;
+
+export interface ToniqDropdownOption {
+    value: unknown;
+    label: string;
+}
+
+export const ToniqDropdown = defineToniqElement({
+    tagName: 'toniq-dropdown',
+    props: {
+        options: [] as Readonly<ToniqDropdownOption[]>,
+        selected: undefined as undefined | Readonly<ToniqDropdownOption>,
+        dropdownOpen: false,
+    },
+    events: {
+        selectChange: defineElementEvent<ToniqDropdownOption>(),
+    },
+    styles: css`
+        :host {
+            ${toniqFontStyles.boldParagraphFont};
+        }
+
+        .dropdown {
+            width: 288px;
+            height: 48px;
+            position: relative;
+            border: ${removeNativeButtonStyles}
+            border-radius: 8px;
+        }
+
+        ${ToniqIcon} {
+            transition: ${interactionDuration} linear;
+        }
+
+        .dropdown.open ${ToniqIcon} {
+            transform: rotate(180deg);
+        }
+
+        .dropdown.open .select-options {
+            display: grid;
+        }
+
+        .dropdown.open .select {
```

Comment:
This also needs to be applied to `.dropdown.open`:
```suggestion
        .dropdown.open .select, .dropdown.open {
```

![border-corner](https://user-images.githubusercontent.com/1205860/172519396-207f4830-ba25-4aae-b84b-e4dcac11c74b.png)